### DISCUSSION TITLE
[2021년 06월 03일] 피어세션 내용

### DISCUSSION THREAD

**Discussion by sglee487** [bcaitech1/p4-mod-model_diet#29] [General]
* loss function인 cosine restart 는 확실히 많이 돌려봐야 하는 것 같다. 올라가다 정체되고 거기서 조금 더 올라가고 하는 방식인것 같아서. 하지만 실시간으로 그래프를 보면서 멈춰있을 때 warmup을 주는게 전에는 좋았었다. 올라가는 경우도 있고 그대로인 경우도 있었다.
* 용우님은 모델 크기 MACs를 천만(10,000,000) 으로 고정 후 learning rate, augment등 다양한 시도를 하고 계심.
* 모델 빨리 학습시키려고 이미지 사이즈 224 -> 64로 했는데 1epoch 시간 3:50 -> 2:17 로 줄음.
